### PR TITLE
Fix #1635 - Add springdoc button

### LINKED ISSUE #1635: Front: add springdoc button

### PR DESCRIPTION
**Pull request by beatfreaker**
Fixes #1635

### CODE REVIEW

File: src/main/webapp/app/springboot/primary/generator/spring-boot-generator/SpringBootGenerator.vue [hunk 1]
```diff
@@ -26,6 +26,7 @@
           :selector-prefix="selectorPrefix"
           @click.prevent="addSpringBootActuator"
         />
+        <GeneratorButtonVue :label="'Spring Doc'" :service="'spring-doc'" :selector-prefix="selectorPrefix" @click.prevent="addSpringDoc" />
```
Comment (Bolo89) [inline src/main/webapp/app/springboot/primary/generator/spring-boot-generator/SpringBootGenerator.vue:29]: Hello, when you click on the spring doc button, it must be colored (in white) when the feature is added on the project. It's missing some code, the service " spring-doc " is not mapped. => Check ServiceProject.ts etc...

**Review by Bolo89 — COMMENTED** [review]
Cypress test must be updated, with the new button  => Generator.spec.ts

File: src/main/webapp/app/springboot/primary/generator/spring-boot-generator/SpringBootGenerator.vue (hunk 1 above)
Comment (beatfreaker) [reply src/main/webapp/app/springboot/primary/generator/spring-boot-generator/SpringBootGenerator.vue:29]: Hi @Bolo89 , handled `spring-doc` service mapping, and now I can see the button turning to white once it is clicked

**Comment by beatfreaker** [comment]
> Cypress test must be updated, with the new button => Generator.spec.ts

Regarding this I have already added test in `Generator.spec.ts` to check if `spring doc` button is getting generated or not, anything else needs to be added too ?

https://github.com/jhipster/jhipster-lite/pull/1733/files#diff-858debb55e6d85be1d68c1e215357e371882b664a2ee7ae3da0245341728825dR41

**Comment by Bolo89** [comment]
> > Cypress test must be updated, with the new button => Generator.spec.ts
> 
> Regarding this I have already added test in `Generator.spec.ts` to check if `spring doc` button is getting generated or not, anything else needs to be added too ?
> 
> https://github.com/jhipster/jhipster-lite/pull/1733/files#diff-858debb55e6d85be1d68c1e215357e371882b664a2ee7ae3da0245341728825dR41

No, I've done à mistake, sorry :/

**Review by Bolo89 — APPROVED** [review]

**Pull request merged**